aim/config.py

```diff
@@ -84,8 +72,12 @@
     cfg.BoolOpt('recovery_restart', default=True,
                 help=("Set to True if you want the agents to exit in critical "
                       "situations.")),
+    cfg.StrOpt('aim_service_identifier', default=socket.gethostname(),
+               help="(NODB) Identifier for this specific AID service, "
```

Comment:
In this is something that is relevant to the end user, the one is may chaneg this option? Should it be in the help string at all? If you really want to put it there, perhaps something more readable like "Not persisted in database" at the end of the text will be better